elementary/gtk-3.0/granite-widgets.css

```diff
@@ -23,6 +23,21 @@
 * Avatar *
 *********/
 
+avatar {
+    color: @SILVER_900;
+    font-weight: 600;
+}
+
+avatar label {
+    background-color: @SILVER_300;
```

Comment:
Would we want to use white-on-blue for now to be more similar to the placeholder face? Ideally eventually I'd like to do a color based on the name hash, maybe from 20 palette colors (light and dark version of each color).

Reply:
tbh I think it doesn't look as good:

![screenshot from 2019-01-03 14 13 34 2x](https://user-images.githubusercontent.com/7277719/50664350-9d7d7500-0f61-11e9-8414-ee1003b617d2.png)

Reply:
@danrabbit the borders and shadows don't look right there. But I'm also mostly indifferent about the color, especially if we plan to do something with programmatic colors down the line. :smile: